testing/unit_tests/fillGapsLargeTests.m

```diff
@@ -2,12 +2,27 @@
 %results = runtests('fillGapsLargeTests.m')
 function tests = fillGapsLargeTests
 tests = functiontests(localfunctions);
+testGurobi = exist('gurobi','file')==3;
+if testGurobi
+    try
+        gurobi_read('solverTests.m');
+    catch
+        testGurobi = false;
+    end
+end
+if ~testGurobi
+    disp('Gurobi not installed or cannot be found in MATLAB path, some fillGapsLargeTests skipped.')
+    skipTests = contains({tests.Name},'gurobi','IgnoreCase',true);
+    tests(skipTests) = [];
+end
+if exist('scip','file')~=3
+    disp('SCIP MEX binary not installed or not functional, some fillGapsLargeTests skipped.')
+    skipTests = contains({tests.Name},'scip','IgnoreCase',true);
+    tests(skipTests) = [];
+end
```

Comment:
I thought the code above was meant to avoid running the SCIP test, which is then to my understanding conflicting with

> Error occurred in solverTests/testSCIP and it did not run to completion.
    ---------
    Error ID:
    ---------
    ''
    --------------
    Error Details:
    --------------
    Error using solverTests>testSCIP (line 77)
    SCIP MEX binary not installed or not functional.
..

If `SCIP MEX binary not installed` all SCIP testing should be skipped, since , the LP solver check already reports `> scip Fail`